docs/diracdoctools/Config.py

```diff
@@ -1,4 +1,7 @@
 """Configuration for the documentation scripts."""
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
This and other changes in diracdoctools are going to conflict with #4571

Reply:
Yes, I avoided the Pilot 2 files (#4162) but any other conflicts will be easy to fix.

When #4571 is merged and propagated I'll rebase this PR.

Reply:
Ok, excellent!